crypto/cipher_extra/aead_test.cc

```diff
@@ -831,6 +831,7 @@ TEST_P(PerAEADTest, AliasedBuffers) {
   EXPECT_EQ(Bytes(kPlaintext), Bytes(in, out_len));
 }
 
+#if !defined(__BIGGEST_ALIGNMENT__) || __BIGGEST_ALIGNMENT__ >= 16
 TEST_P(PerAEADTest, UnalignedInput) {
   alignas(16) uint8_t key[EVP_AEAD_MAX_KEY_LENGTH + 1];
```

Comment:
Could this test use alignas(8) and still cover the case of unaligned input because the test uses all the pointers + 1 to get unaligned like  `ciphertext + 1`?

Reply:
You're right. I changed the test to just use the `__BIGGEST_ALIGNMENT__` value if available.